### ISSUE TITLE
Can't send/receive events between two iPhones

### ISSUE THREAD

**Issue by dmdque** [pusher/pusher-websocket-swift#99]
I have three clients running: one iPhone simulator client, one physical iPhone client, and one web client. (Both iPhone clients were loaded with XCode.) I also have a rails server configured with the key, secret, and app_id.

I'm able to broadcast an event from the server to all three clients. I'm also able to broadcast from either iPhone to the web client. But for some reason, the two iPhone clients can't send/receive events from each other.

I've set up authentication and am triggering the event `client-my_event` to the channel `private-test_channel`. I can't figure out why two of the clients can't communicate with each other, when they're able to send and receive events on the same channel/event.

I'm not sure what additional information I can provide to help debug this, so let me know if you have any hunches.

**Comment by hamchapman** [comment]
Could you provide a sample project that displays this problem?

If not, please could you provide information about the specifics of the setup that produces this problem: version of Xcode, version of iOS, which device is being used to test (e.g. iPhone 7, 6S etc), which version of PusherSwift?

Thanks!

**Comment by dmdque** [comment]
Thanks for replying so quickly Hamilton. I've set up a project here: https://github.com/dmdque/PusherClientExample

I'm using [Slanger](https://github.com/stevegraham/slanger) as the backend in my project, and it turns out that's the source of the problem. Once I switched to using Pusher as the backend, it worked.

I'll open an issue for the Slanger project, but if you've got any insight I'd love to hear it!

Thanks.
